### ISSUE TITLE
When I make a mod pack on Curse Forge, the game boots up and crashes when I open a world. [1.16.5]

### ISSUE THREAD

**Issue by ObriWanCanobro** [Iskallia/Vault-public#984]
Mod's all work together in Minecraft, but after putting the mods into a Curse Mod pack the game no longer allows me to play on a world through the pack profile. When waiting for the world to load, the screen turns white, after clicking or waiting long enough I get the Minecraft no longer responding thing and if I wait for a response it crashes. been trying to figure this out all day.
I am using Forge 36.2.34 
Crash log and Crash report below, Thank you.

[latest.log](https://github.com/Iskallia/Vault-public-S1/files/8932849/latest.log)

[crash-2022-06-17_22.26.49-client.txt](https://github.com/Iskallia/Vault-public-S1/files/8932850/crash-2022-06-17_22.26.49-client.txt)

**Comment by bmemike** [comment]
Can you explain what you mean by "when you make a mod pack"? 

If you're trying to incorporate the the_vault mod into a new modpack, the devs ask that you do not. 

In addition to that, it's completely integrated with the mods in the vault hunters modpack and you're not going to be able to use it without all of those other mods. It was never intended to be a stand-alone mod.

**Comment by ObriWanCanobro** [comment]
The_vault isn't in my mod pack, so I'm not sure what you mean by this. 

Basically if I install mods one by one without curse forge, through the .Minecraft/mods folder then it works fine on my PC and in my worlds, but if I create the mod pack (same mods and versions) in the Curse Forge profile, it doesn't work correctly, if I attempt to open a world it crashes as seen above. I'm trying to make all the mods I've used during the build process into a mod pack for the world I've been building so others can play it, but it will not work with any method I've used so far through curse. 

Unfortunately due to legality issues and the amount of players backing this project I can't use a zipped folder with MediaFire, Just because you're not allowed to redistribute most mods. So I need help fixing this
